### PR TITLE
Small strange jackboots buff

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

Makes jackboots reduce slowness from SlowOnDamage by 50%.

In a practical sense:
- Light damage (>40) goes from a 30% slowdown to a 15% slowdown.
- Heavy damage (>80) goes from a 50% slowdown to a 25% slowdown.

Removes combat boots from the sec loadout as this makes them objectively worse and turns them into a silly noob trap.

Credit to Bhijn & Myr for the idea

## Why / Balance

This PR is primarily aimed to help reduce the prevalence of traitor contraband abuse from security players.

NoSlips are one of the most over represented items when it comes to syndicate contraband being illegally used by sec. This is largely because of a distinct power vacuum when it comes to footwear: there are basically only 3 options commonly available
- Regular shoes, which are legal and everyone has by default
- Magboots, which are legal but have a shitty combat downside (reduced speed) in exchange for the antigrav and noslip properties.
- NoSlips/galoshes, illegal or unobtainable but have a situationally good combat upside (specifically in the wake of a lot of slip nerfs)

Sec themselves (the only ones who are particularly relevant in terms of contraband abuse in this situation) have no mechanical reason to _not_ use noslips, even though the rules prohibit it and the upside is relatively minor. Compared to no upside or even a combat downside, noslips are an obvious choice.

My solution for this--this PR--is to buff jackboots with a _generally useful_ upside to serve as a natural alternative to the _situational utility_ of noslips. Reducing sec's slowdown in combat situations when injured is great for allowing them to carry on fighting in tense situations as well as escaping ambushes. 

I don't think this is a 100% silver bullet to the entire situation, but my hope is that it'll serve as an alternative that will at least greatly reduce sec's usage of syndicate equipment, or at the very least, make the usage of said equipment into an interesting tactical decision rather than a flagrant showing-off of contraband.

## Technical details

Adds a new `ClothingSlowOnDamageModifierComponent` and some events to hook into `SlowOnDamageSystem`.

## Media

image

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Jackboots now reduce slowness from injuries by 50%.
- remove: Removed combat boots from the security loadout.

### CODE REVIEW

**Comment by Hanzdegloker** [comment]
Love the idea, but any particular reason it applies only to jackboots and not to combat boots as well?

**Comment by EmoGarbage404** [comment]
> Love the idea, but any particular reason it applies only to jackboots and not to combat boots as well?

Jackboots are only found in sec whereas combat boots are worn by nukies and syndies.

**Comment by Cojoke-dot** [comment]
GUAH!!! Sec winter boots my beloved, they will never get the love of the other sec boots 😞

**Comment by Hanzdegloker** [comment]
> > Love the idea, but any particular reason it applies only to jackboots and not to combat boots as well?
> 
> Jackboots are only found in sec whereas combat boots are worn by nukies and syndies.

Sounds about right, Sec buff is real then

**Comment by bruhmogus** [comment]
i feel like we should just buff combat boots as well but make syndies get a different version that doesn't have the buff
(style simulator 14 my beloved)

**Comment by DieselMohawk** [comment]
Combat boots come with knives while jackboots often do not. Will officers come with knives if they choose the jackboots?

**Comment by Catdere** [comment]
Ill be real.. this is the worst thing ive ever seen suggested ever.. BUT on the other hand this fixes the issue of sec powergaming
and gives a real alternative to noslips.. But why not also give this to combat boots?!? allowing Nukies to take advantage of the movement speed

**Comment by Cojoke-dot** [comment]
> Combat boots come with knives while jackboots often do not. Will officers come with knives if they choose the jackboots?

The loadout for all current sec boots has knives, including the winter pair. In addition, the SecVend has extra knives.

**Comment by deltanedas** [comment]
nukies not buying noslips is instant flukie no reason to give it to combat boots

**Comment by Cojoke-dot** [comment]
> nukies not buying noslips is instant flukie no reason to give it to combat boots

No reason to take combat boots from sec and not just buff them too then

File: Content.Shared/Damage/Components/ClothingSlowOnDamageModifierComponent.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Damage.Components;
+
+/// <summary>
+/// This is used for a clothing item that modifies the slowdown from taking damage.
```
Comment (SlamBamActionman) [inline Content.Shared/Damage/Components/ClothingSlowOnDamageModifierComponent.cs:6]: Can't find anything wrong with the code, it looks solid to me. I recommend adding a `<see cref="SlowOnDamageComponent"/>` reference to the component's summary though, just to make it explicitly clear what values it is modifying.

**Review by SlamBamActionman — APPROVED** [review]

**Comment by SlamBamActionman** [comment]
I don't think it's necessary to add this buff to the Sec winter boots by the way. The benefit they provide is the temperature resistance and makes it a fair trade-off if Sec needs to fight in adverse temperature conditions.

**Comment by ps3moira** [comment]
DO YOU WANT TO WEAR A MACHO UNIFORM!~♪

**Comment by UbaserB** [comment]
I generally like this idea

**Comment by LordEclipse** [comment]
Based

**Pull request merged**